<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lobbyJoinView</class>
 <widget class="QWidget" name="lobbyJoinView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #8A8557;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>109</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Joined Gamelobby</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hostGameButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>200</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: grey;
border-color: #111111;
    

</string>
   </property>
   <property name="text">
    <string>Spiel leiten</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="joinGameButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>250</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
border-color: #111111;

</string>
   </property>
   <property name="text">
    <string>Spiel beitreten</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="leaveLobbyButton">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>300</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
border-color: #111111;
   
</string>
   </property>
   <property name="text">
    <string>Lobby verlassen</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatField">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>350</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatMsgTextField">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>609</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMsgButton">
   <property name="geometry">
    <rect>
     <x>871</x>
     <y>609</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Senden</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Deine IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Spielnummer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipAdressLbl">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>260</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>255.255.255.255</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameNumberLbl">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>280</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>1234567890123456789</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setRdyButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>610</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: grey;
color: red;</string>
   </property>
   <property name="text">
    <string>Nicht Bereit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="playerList">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none</string>
   </property>
  </widget>
  <widget class="QListWidget" name="rdyList">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>51</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;</string>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
